frontend/src/utils/utils.ts

```diff
@@ -63,6 +64,27 @@ export function partition<T>(
   );
 }
 
+/**
+ * @function excludeMetaTag
+ * Filter out a configured list of select metadata or tags from a COMS response
+ * @param {object} type either 'metadata' or 'tagset'
+ * @param {object} axiosResponse A resolved axios response from COMS
+ * @returns {object} The response data with select metadata/tags from a configured list removed
+ */
+export function excludeMetaTag(type: string, data: any){
+  // array of selected tags/metadata (keys) to hide from UI
+  const excludeArray = new ConfigService().getConfig().exclude?.[type].split(',') ?? [];
+  // filter COMS data
+  return data.map((obj: any) => {
+    return {
+      ...obj,
+      [type]: obj[type]?.filter((el: any) => {
+        return !excludeArray.includes(el.key);
+      })
+    };
+  });
```

Comment:
We can leverage closures and inlining to reduce the amount of explicit return statements needed for the operation.
```ts
// Exclude specific values
return data.map((obj: any) => ({
  ...obj,
  [type]: obj[type]?.filter((el: any) => !excludeArray.includes(el.key))
}));
```